calico-cloud/compliance/enable-compliance.mdx

```diff
@@ -0,0 +1,42 @@
+---
+description: Enable compliance to configure reports to assess compliance for all assets in a Kubernetes cluster.
+---
+
+# Enable Compliance
```

Comment:
This title doesn't sound quite right to me, but maybe I'm just not familiar with the jargon. "Compliance" sounds like a large, abstract concept. The function of this set of features is to enable reporting that lets you figure out whether your system complies with various standards, right? 

If I'm not off here, we should go with a title along the lines of "Enable compliance reporting".

Reply:
We should change instances of 'compliance' to 'compliance reporting' throughout, where applicable.

Reply:
compliance include compliance reports, compliance snapshot and compliance benchmark. How able using "compliance features" instead @ctauchen

Reply:
I get that there's a specific component that deals with reports, while others do other things. But I still think the general idea here is all in service of the compliance reports, no?

And in any case, we should continue to match the UI, which is clearly about enabling compliance reports, not 'compliance' or 'compliance features'. 

We can open the conversation wider if necessary, but I'm pretty confident we should just change these to 'compliance reports' throughout.

Reply:
FYI @vara2504